app/_src/kubernetes-ingress-controller/upgrade/kic.md

```diff
@@ -73,6 +73,33 @@ complete the following steps to upgrade to version v1.1 of the CRD:
 
 {% endif_version %}
 
+{% if_version gte:3.4.x %}
+
+### Update Gateway API from v1.1 to v1.2
+
+Starting from version 3.4, {{ site.kic_product_name }} supports Gateway API version 1.2.
+There is a breaking change in gateway API 1.2 to remove the `v1alpha2` version of `GRPCRoute` and `ReferenceGrant`.
+
+If you have been using gateway API v1.1 and {{ site.kic_product_name }} 3.2 and above, and there are no `GRPCRoute` and `ReferenceGrant` resources stored in `v1alpha2` version, you can directly upgrade gateway API from v1.1 to v1.2. 
+You can use the following script to ensure your `GRPCRoute` and `ReferenceGrant` CRDs are not using `v1alpha2` storage version:
+
+```bash
+kubectl get grpcroutes -A -o jsonpath='{.items[*].apiVersion}' | tr ' ' '\n' | sort | uniq -c
+kubectl get referencegrants -A -o jsonpath='{.items[*].apiVersion}' | tr ' ' '\n' | sort | uniq -c
+```
+If any of the output contains `v1alpha2`, it means that there are `GRPCRoute` or `ReferenceGrant` (or both) using `v1alpha2` storage version and you need to update the manifests before upgrading.
```

Comment:
```suggestion

If the output contains resources in `v1alpha2` version, it means that there are `GRPCRoute` or `ReferenceGrant` (or both) using `v1alpha2` storage version and you need to update the manifests before upgrading.
```

Reply:
The script will only output and collect the API version of resources so the result of the script would be:
```
    10 gateway.networking.k8s.io/v1
      2 gateway.networking.k8s.io/v1alpha2
```
So I think I would keep the current words.